### DISCUSSION TITLE
为什么首页的锚点不在最前面，怎么改？

### DISCUSSION THREAD

**Discussion by lomiles** [halo-dev/halo#5918] [General]
刚搭建了1个官网，用的是Daisyblocks 1.5.1主题，地址是http://www.tengogo.com/

可以进入看看，每次进去不是最前面的头图块，而是最下面的表单收集
![image](https://github.com/halo-dev/halo/assets/35911812/cafb7eb9-85e3-421e-a487-184daf0e0612)

但如果把地图设置删掉，就是正常的了。难道不能设置地图链接（可模板预留了大块空白，没选择啊）

**Comment by wan92hen** [comment]
百度地图的这个链接地址，可能内置了自动聚焦相关的代码，目前还没有找到比较好的处理方式，使用谷歌地图的分享链接目前是没有这样的 问题。

**Reply by lomiles** [reply]
建议增加1个样式选项，去掉地图代码，改用一张图片占位
